learn/_pages/LandingPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { LearningList } from "../_components/LearningList.tsx";
+import { sidebar } from "../tutorials/_data.ts";
+import { TutorialIcon } from "../_components/TutorialIcon.tsx";
+import { ExampleIcon } from "../_components/ExampleIcon.tsx";
+import { VideoIcon } from "../_components/VideoIcon.tsx";
+
+export default function LandingPage() {
+  const componentsPerSidebarItem = sidebar.map(
+    (item: { title: string; items: any[] }) => {
+      return (
+        <LearningList
+          title={item.title}
+          items={item.items}
+        />
+      );
+    },
+  );
+
+  return (
+    <main
+      id="content"
+      className="flex flex-col px-8 pt-6 md:pt-12 mt-4 md:items-center md:justify-center max-w-[1200px] mx-auto mb-48"
+    >
+      <div className="w-full mb-6 max-w-screen-xl mx-auto hub-header">
+        <div className="max-w-prose mb-8">
+          <h1 className="text-2xl font-semibold sm:text-3xl md:text-4xl mb-3">
+            Learning hub
+          </h1>
+          <p>
+            A collection of walkthrough tutorials, examples, videos and guides
+            to teach you about the Deno runtime and how to use it with your
+            favorite tools.
+          </p>
+        </div>
+        <div className=" mb-2 flex gap-2 flex-wrap items-center">
+          <h2 className="font-semibold mb-2 mt-2">
+            Filter by type:
+          </h2>
+          <label for="example" className="mr-4 ml-4 flex items-center">
+            <ExampleIcon color="#9d9d9d" />Examples:
+            <span className="switch">
+              <input
+                type="checkbox"
+                id="example"
+                value="Examples"
+                className="mr-2"
+                checked
+              />
+              <span className="slider"></span>
+            </span>
+          </label>
```

Comment:
Issue: there's no accessible text for these labels, and no visible focus indication.

Also: ideally, these toggles should be grouped somehow, probably by a `<form>` element.

Reply:
Is the accessible text not the text in the label?